mmf/models/mmbt.py

```diff
@@ -7,24 +7,34 @@
 
 import os
 from copy import deepcopy
-from typing import Dict, Optional
+from dataclasses import dataclass
+from typing import Dict, Optional, Union
 
 import torch
 from mmf.common.registry import registry
 from mmf.models.base_model import BaseModel
 from mmf.models.interfaces.mmbt import MMBTGridHMInterface
-from mmf.modules.encoders import MultiModalEncoderBase
+from mmf.modules.encoders import (
+    Encoder,
+    ImageEncoder,
+    ImageEncoderTypes,
+    MultiModalEncoderBase,
+    ResNet152ImageEncoder,
+    TextEncoder,
+    TextEncoderTypes,
+    TransformerEncoder,
+)
 from mmf.modules.hf_layers import replace_with_jit
 from mmf.utils.checkpoint import load_pretrained_model
 from mmf.utils.configuration import get_mmf_cache_dir
 from mmf.utils.modeling import get_optimizer_parameters_for_bert
-from omegaconf import OmegaConf
+from omegaconf import II, DictConfig, OmegaConf
 from torch import Tensor, nn
 from transformers.modeling_bert import BertForPreTraining, BertPredictionHeadTransform
 
 
 # TODO: Remove after transformers package upgrade to 2.5
-class MMBTConfig:
+class MMBTConfigForTransformers:
```

Comment:
We should keep this as `MMBTConfig`. These were copied from Huggingface and once we bump up the internal version we can replace these without changes to other parts of the code.

Reply:
Makes sense.